docs/book/src/tasks/healthcheck.md

```diff
@@ -116,10 +140,9 @@ Explicit skipping using `cluster.x-k8s.io/skip-remediation` annotation:
 
 Before deploying a MachineHealthCheck, please familiarise yourself with the following limitations and caveats:
 
-- Only Machines owned by a MachineSet will be remediated by a MachineHealthCheck
-- Control Plane Machines are currently not supported and will **not** be remediated if they are unhealthy
+- Only Machines owned by a MachineSet will be remediated by a MachineHealthCheck (since a MachineDeployment uses a MachineSet, then this includes Machines that are part of a MachineDeployment)
```

Comment:
should this say
```suggestion
- Only Machines owned by a MachineSet or a KubeadmControlPlane can be remediated by a MachineHealthCheck (since a MachineDeployment uses a MachineSet, then this includes Machines that are part of a MachineDeployment)
```